python/servo/testing_commands.py

```diff
@@ -226,6 +227,32 @@ def test_content(self):
     @CommandArgument('--no-progress', default=False, action="store_true",
                      help="Don't show progress for tidy")
     def test_tidy(self, all_files, no_progress):
+        def check_dep_license_errors() -> bool:
```

Comment:
Could this actually move to `tidy.py`? You're right that we have a long term goal of not longer relying on `mach` an the idea is that `tidy.py` can be used standalone.

Reply:
Moved the function to tidy.py. I still kept it separate from the scan() function, since cargo deny operates on crates, not files.